src/modules/expense/Expense.tsx

```diff
@@ -0,0 +1,47 @@
+import { Stack, Button } from '@mui/material';
+import { Paper } from '../../shared-components/Paper/Paper';
+import { styled, Theme } from '@mui/material/styles';
+import { InfoBar } from '../../shared-components/InfoBar';
+import { Header } from './Header';
+import { PriceSection } from './PriceSection';
+import { Form } from './Form';
+
+const StyledDiv = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  justifyContent: 'space-between',
+  minHeight: `calc(100vh - ${theme.spacing(3)})`,
+  backgroundColor: '#FD3C4A',
+}));
+
+const StyledDivUpperSide = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  flexDirection: 'column',
+  backgroundColor: '#FD3C4A',
+  minHeight: '300px',
+}));
+
+const StyledDivBottomSide = styled(`div`)(({ theme }: { theme: Theme }) => ({
+  display: 'flex',
+  backgroundColor: '#FAF9F6',
+  minHeight: '600px',
+  marginBottom: '0',
+  borderRadius: '32px 32px 0 0',
+}));
+
+export const Expense = () => {
+  return (
+    <Paper>
+      <StyledDiv>
+        <StyledDivUpperSide>
+          <InfoBar />
+          <Header />
+          <PriceSection />
+        </StyledDivUpperSide>
+        <StyledDivBottomSide>
```

Comment:
This element would need some padding so that there's some space between the form and the edge of the sreen.